crates/sui-indexer/src/models/checkpoints.rs

```diff
@@ -42,6 +42,8 @@ pub struct StoredCheckpoint {
     pub checkpoint_commitments: Vec<u8>,
     pub validator_signature: Vec<u8>,
     pub end_of_epoch_data: Option<Vec<u8>>,
+    pub min_tx_sequence_number: Option<i64>,
```

Comment:
I remember us having a discussion regarding this under a PR and it was related to some checkpoints containing no txns in testing setup, but why does this need to be `Option` again? @wlmyng

Reply:
I think we can privilege @gegaowp 's implementation over the stuff here (iirc his changes were merged into main instead of the breaking changes park)

Reply:
Actually we'll need to keep these min/max tx_sequence_numbers the way they are because the backfill dbs have uses schema and Ge's implementation for pruner actually didn't add these columns in `checkpoints` table.